src/fileIO.cpp

```diff
@@ -523,6 +540,11 @@ json Module::toJson() {
 json Generator::toJson() {
   json j = Instantiable::toJson();
   j["genparams"] = Params2Json(genparams);
+    cout << "AM I HERE0??" << endl;
```

Comment:
Seems like a debugging message, can we silence these with a comment? (Or perhaps add a debug_print macro that's off by default)